conf/config.yml

```diff
@@ -270,6 +272,8 @@ auth:
   #     iterations: 100
   #     # the length of the resulting key
   #     keyLength: 32
+  #     # fail authentication process if invalid login parameters are used
+  #     # reportInvalidParameters: true
```

Comment:
looking at the code here, it would make sense the config is passed into the auth strategies via the combiner

i can't see a situation where we would want the reportInvalidParameters to be true in a combined and false normally.

Reply:
So if there is more than one auth the combiner should pass the reportInvalidParameters: false to the isValid function ?

Reply:
yup